index.html

```diff
@@ -1,12 +1,105 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to Evyy Cafe!</title>
+    <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:wght@400;700&family=Playfair+Display:wght@400;700&family=Quicksand:wght@400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Cabin+Sketch:wght@400;700&display=swap" rel="stylesheet">
+    <link href="style/style.css" rel="stylesheet">
+    <link href="style/index.css" rel="stylesheet">
 </head>
-<body>
-  
+<body class="home-page">
+    <header id="evyy">
+        <div class="header-content">
+            <img id="logo" src="images/evyycafelogo.png" alt="Evyy Cafe Logo - A pastel pink cafe logo">
+            
+            <nav class="navigation">
+                <ul class="main-nav"> 
```

Comment:
There are places across the project where classes are declared when we could use the elements themselves. For example, the only `nav` elements in the project are these in the headers on each page. This means we could:
- remove the classes on these lines
- replace `nav.navigation` with `nav` in the relevant CSS rules
- replace `.main-nav` with `nav ul` in the relevant CSS rules

We want to create classes when they make our lives easier or solve a problem, but there are many cases where they aren't necessarily benefitting us. 
- Taking a look across the project, what other classes or ids do you see that could be refactored to simplify the code and/or rules?